src/api/call-api.ts

```diff
@@ -1,67 +1,28 @@
-import * as adapter from '@api3/airnode-adapter';
```

Comment:
Do we even still need this file? I was hoping we would get rid of it once we use the `airnode-node` version.

Reply:
The idea was to use this as a centralized place to construct the node.handlers.callApi inputs instead of doing these inline in the handlers since they work a bit differently as well as parse the result value (see [here](https://github.com/api3dao/airkeeper/pull/44#discussion_r829983119))

Please let me know if you'd prefer to change this

Reply:
Alright, makes sense :slightly_smiling_face: